### ISSUE TITLE
Jedi integration

### ISSUE THREAD

**Issue by CarlAndersson** [mandeep/sublime-text-conda#17]
Would it be possible to make the jedi plugin (https://packagecontrol.io/packages/Jedi%20-%20Python%20autocompletion) use the currently activated environment?
Jedi can use of build system variables to set the path. Are you exposing the conda environment to other build systems?

**Comment by mandeep** [comment]
I don't believe I am exposing the build system. If you have a link, I'll gladly look into it.

**Comment by CarlAndersson** [comment]
If you have a look under the "Settings" header on https://packagecontrol.io/packages/Jedi%20-%20Python%20autocompletion, you can see how Jedi is normally configured. In the example they use `$project_path` as part of the project-level configuration.
It would be super if there was a way to use e.g. `$conda_env` to configure other packages.
